internal/deb/extract.go

```diff
@@ -153,11 +134,7 @@ func extractData(dataReader io.Reader, options *ExtractOptions) error {
 			return err
 		}
 
-		sourcePath := tarHeader.Name
-		if len(sourcePath) < 3 || sourcePath[0] != '.' || sourcePath[1] != '/' {
-			continue
-		}
-		sourcePath = sourcePath[1:]
+		sourcePath := sanitizeTarPath(tarHeader.Name)
```

Comment:
Thanks for moving this out. Can we please have an explicit `ok` result instead of expecting the call site to know it must check the result?

Reply:
Done in https://github.com/canonical/chisel/pull/179/commits/321bacea7bc18935a1ef80ac6d21a01608ecac6c.